setup/install/index.md

```diff
@@ -10,20 +10,23 @@ This section describes how to install and set up Spinnaker so that it can be con
 use in production. If you just want to evaluate Spinnaker without much work, one of the options
 in [Quickstart](/setup/quickstart/) might be a better choice.
 
+## What you'll need
+
+* A machine on which to install Halyard
+
+  This can be a local machine or VM (Ubuntu 14.04/16.04, Debian, or macOS), or
+  it can be a Docker container.
+
+* A Kubernetes cluster on which to install Spinnaker itself
+
 ## The process
 
 Installing a complete Spinnaker involves these steps:
 1. [Install Halyard](/setup/install/halyard/)
 2. [Choose a cloud provider](/setup/install/providers/)
 3. [Choose an environment](/setup/install/environment/)
 4. [Deploy Spinnaker](/setup/install/deploy/)
-
-
-
-## Next steps
-
-Let's start by [setting up Halyard](/setup/install/halyard/), which will manage
-the configuration, installation, and updates of Spinnaker for you.
-
-After this process is done, you can use Spinnaker to create pipelines and deploy software, 
-but there are some [further configuration steps](/setup/other_config/) you're likely to need.
+5. [Configure everything else](/setup/other_config/) (which includes a lot of
+  stuff you need before you can use Spinnaker in production)
+6. [Productionize Spinnaker](/setup/productionize/) (which mainly helps you
+  configure Spinnaker to scale for production)
```

Comment:
Certainly not something to address in this PR, but I think it's a little confusing that "configure everything else" and "productionize Spinnaker" are separate sections that seem to both be required to run Spinnaker in production.

Reply:
Yeah, I was just noticing that.

Reply:
@lwander , what if "Productionize Spinnaker" were moved under "Configure Everything Else"?

Reply:
I had it at the top-level since it feels like a large section, is immediately visible to anyone curious about Spinnaker & how it scales. It's also thematically a little different since it requires more understanding of Spinnaker than the other sections, and will likely require gathering & reading your own metrics/logs before deciding what to do.

Reply:
Fair enough. We'll have to come up with some other way of dealing with the fact that "configure everything else" isn't really everything else.

Reply:
What if we changed it to something like "set up remaining services"  instead of "configure everything else"? I think that would make it a little clearer that we're dividing the docs into a first pass Spinnaker installation, thinking about things like metrics/security/etc, and finally a fully production-ready instance.

Reply:
+1 to separate top-level section for productionize. We get enough inquiries in this realm, and it's useful toward emphasizing the battle-hardening we have. Also, generally I support the notion that Nora is bringing up, but this feels pretty non-controversial to solve.